src/js/tests/bindings.test.js

```diff
@@ -0,0 +1,73 @@
+import addon from '../lib/bindings.cjs';
+
+import assert from 'node:assert';
+import { describe, it, before, after } from 'node:test';
+import { models } from './models.js';
+
+const MODEL_PATH = process.env.MODEL_PATH
+  || `./tests/models/${models[0].split('/')[1]}`;
+
+describe('bindings', () => {
+  let pipeline = null;
+
+  before((_, done) => {
+    pipeline = new addon.LLMPipeline();
+
+    pipeline.init(MODEL_PATH, 'AUTO', (err) => {
```

Comment:
```suggestion
    pipeline.init(MODEL_PATH, 'CPU', (err) => {
```
Setting that to CPU allows to avoid compiling all other plugins saving some CI time.